CHANGELOG.md

```diff
@@ -1,7 +1,18 @@
 Changelog
 =========
 
-New Feature
+New Features
+
+- Bounce integral methods with ``desc.integrals.Bounce1D`` and ``desc.integrals.Bounce2D``.
```

Comment:
Ok in that case, can you also take a look at the docstring for `Bounce1D` and 2d and let me know if you prefer to make anything a regular comment instead of a public doc